lib/std/os/linux.zig

```diff
@@ -6146,7 +6146,242 @@ pub const termios = switch (native_arch) {
     },
 };
 
+// Routing table calls.
+/// Add routing table entry
+pub const SIOCADDRT = 0x890B;
+
+/// Delete routing table entry
+pub const SIOCDELRT = 0x890C; 
+
+/// Unused
+pub const SIOCRTMSG = 0x890D; 
+
+// Socket configuration controls.
+/// Get iface name
+pub const SIOCGIFNAME = 0x8910;
+
+/// Set iface channel
+pub const SIOCSIFLINK = 0x8911;
+
+/// Get iface list
+pub const SIOCGIFCONF = 0x8912; 
+
+/// Get flags
+pub const SIOCGIFFLAGS = 0x8913;
+
+/// Set flags
+pub const SIOCSIFFLAGS = 0x8914; 
+
+/// Get PA address
+pub const SIOCGIFADDR = 0x8915; 
+
+/// Set PA address
+pub const SIOCSIFADDR = 0x8916; 
+
+/// Get remote PA address
+pub const SIOCGIFDSTADDR = 0x8917; 
+
+/// Set remote PA address
+pub const SIOCSIFDSTADDR = 0x8918; 
+
+/// Get broadcast PA address
+pub const SIOCGIFBRDADDR = 0x8919;
+
+/// Set broadcast PA address
+pub const SIOCSIFBRDADDR = 0x891a; 
+
+/// Get network PA mask
+pub const SIOCGIFNETMASK = 0x891b; 
+
+/// Set network PA mask
+pub const SIOCSIFNETMASK = 0x891c;
+
+/// Get metric
+pub const SIOCGIFMETRIC = 0x891d; 
+
+/// Set metric
+pub const SIOCSIFMETRIC = 0x891e;
+
+/// Get memory address (BSD)
+pub const SIOCGIFMEM = 0x891f;
+
+/// Set memory address (BSD)
+pub const SIOCSIFMEM = 0x8920; 
+
+/// Get MTU size
+pub const SIOCGIFMTU = 0x8921; 
+
+/// Set MTU size
+pub const SIOCSIFMTU = 0x8922; 
+
+/// Set interface name
+pub const SIOCSIFNAME = 0x8923; 
+
+/// Set hardware address
+pub const SIOCSIFHWADDR = 0x8924;
+
+/// Get encapsulations
+pub const SIOCGIFENCAP = 0x8925;
+
+/// Set encapsulations
+pub const SIOCSIFENCAP = 0x8926;
+
+/// Get hardware address
+pub const SIOCGIFHWADDR = 0x8927;
+
+/// Driver slaving support
+pub const SIOCGIFSLAVE = 0x8929;
+
+/// Driver slaving support
+pub const SIOCSIFSLAVE = 0x8930;
+
+/// Add to Multicast address lists
+pub const SIOCADDMULTI = 0x8931;
+
+/// Delete from Multicast address lists
+pub const SIOCDELMULTI = 0x8932;
+
+/// name -> if_index mapping
 pub const SIOCGIFINDEX = 0x8933;
+
+/// Set extended flags set
+pub const SIOCSIFPFLAGS = 0x8934; 
+
+/// Get extended flags set
+pub const SIOCGIFPFLAGS = 0x8935;
+
+/// Delete PA address
+pub const SIOCDIFADDR = 0x8936; 
+
+/// Set hardware broadcast addr
+pub const SIOCSIFHWBROADCAST = 0x8937; 
+
+/// Get number of devices
+pub const SIOCGIFCOUNT = 0x8938;
+
+/// Bridging support
+pub const SIOCGIFBR = 0x8940;
+
+/// Set bridging options
+pub const SIOCSIFBR = 0x8941;
+
+/// Get the tx queue length
+pub const SIOCGIFTXQLEN = 0x8942; 
+
+/// Set the tx queue length
+pub const SIOCSIFTXQLEN = 0x8943; 
+
+// These 2 were commented in sockios.h not sure if it's needed or not.
+// pub const SIOCGIFDIVERT = 0x8944		Frame diversion support
+// pub const SIOCSIFDIVERT = 0x8945		Set frame diversion options
```

Comment:
These 2 were commented in sockios.h not sure if it's needed or not.

Reply:
If the UAPI headers don't define them, we shouldn't either, so you can just remove these.